tmoistner/Lab4.py

```diff
@@ -0,0 +1,30 @@
+__author__ = 't'
+
+
+"""does magic tricks"""
+
+
+def reverse(i):
+    """reverses number"""
+    return int(str(i)[::-1])
+
+
+def descending(i):
+    """checks that number is descending"""
+    largest = 10
+    for c in str(i):
+        x = int(c)
+        if x >= largest:
+            return False
+        else:
+            largest = x
+    return True
+
+
+def magic():
```

Comment:
move this function to the test function and separate out the trick from the testing of the trick.